src/rust/lib/frp/src/macros.rs

```diff
@@ -68,35 +84,35 @@ macro_rules! new_bridge_network {
 
 /// Extends the provided network with new rules. See documentation of `new_network` to learn more.
 #[macro_export]
-macro_rules! _extend_network {
+macro_rules! _extend {
```

Comment:
The documentation for `_extend` and `extend` is the same. Please add note which will distinct those two.